log-postgres/log-postgres.cabal

```diff
@@ -1,5 +1,5 @@
 name:                log-postgres
-version:             0.7
+version:             0.7.0.1
```

Comment:
Do we need to bump version in `log` too, or could we deprecate the `log` package? re-exports are tricky from PVP pov.